x-pack/plugins/apm/server/routes/service_groups/get_services_counts.ts

```diff
@@ -67,10 +67,14 @@ export async function getServicesCounts({
 
   const buckets: Record<string, { services_count: { value: number } }> =
     response?.aggregations?.service_groups.buckets ?? {};
-  return Object.keys(buckets).reduce((acc, key) => {
+
+  const servicesCount: Record<string, number> = Object.keys(buckets).reduce<
```

Comment:
Why store the result in a variable if you're not using it, can't you just return it? And type the function return?